### ISSUE TITLE
Not working for `pages` router

### ISSUE THREAD

**Issue by heymartinadams** [uixmat/onborda#14]
I’m one (of many) devs who are laggards and have not adopted the `app` router.

When I install `onboarda` for the `pages` router, I run into the following issue: `params` is `null` and therefore throws an exception.

```ts
const params = useParams<{ step: string }>();
...
useEffect(() => {
    if (isOnbordaVisible) {
      console.log("Onborda: Initialising...");
      if (params.step) {
        setCurrentStep(parseInt(params.step));
      }
      if (steps.length > 0) {
        const firstStepElement = document.querySelector(steps[0].selector);
        if (firstStepElement) {
          setPointerPosition(getElementPosition(firstStepElement));
        }
      }
    }
  }, [params.step, steps, isOnbordaVisible]);
```

Please change to ` if (params?.step) {` and `[params?.step, ...]`.

https://github.com/uixmat/onborda/blob/5a4065462050929c51c9bf6ff6d70fbd6027978e/src/Onborda.tsx#L38-L51

**Comment by BankkRoll** [comment]
:heavy_plus_sign:1

Same here i messed for hours, ended up building my own insite directly based of onborda. Im a pages router kinda guy myself.. 

Tip: Use dynamic imports to get anything imported. But for me nothing seemed to work even then.

**Comment by realtydev** [comment]
i rewrote the lib for this reason plus needed better element observability for dynamic pages.

**Comment by uixmat** [comment]
This should be fixed in version 1.2.3, will close the issue but feel free to reopen and tag me in a new issue if you guys still have problems!
